app/controllers/concerns/tokens_management.rb

```diff
@@ -0,0 +1,36 @@
+module TokensManagement
+  extend ActiveSupport::Concern
+
+  def extend_token
+    authorize @token, :extend?
+
+    render 'shared/tokens/extend'
+  rescue StandardError
+    redirect_current_user_to_homepage
+  end
+
+  def ask_for_extension
```

Comment:
```suggestion
  def ask_for_prolongation
```

si on reste consistant avec le reste